Sources/Core/Shared/OperationQueue.swift

```diff
@@ -197,7 +197,7 @@ public extension OperationQueue {
 
      - returns: The main queue
      */
-    public override class func main() -> OperationQueue {
+    public class func main() -> OperationQueue {
```

Comment:
It's 
```
public extension Operations.OperationQueue {

    private static let sharedMainQueue = MainQueue()

    public override class var main: Operations.OperationQueue {
        Return sharedMainQueue
    }
}
```

now, because main was changed to be a class var instead of a class func.